packages/panels/resources/views/components/tenant-menu.blade.php

```diff
@@ -91,9 +91,10 @@ class="ms-auto h-5 w-5 shrink-0 text-gray-400 transition duration-75 group-hover
                 <x-filament::dropdown.list.item
                     :color="$profileItem?->getColor()"
                     :href="$profileItemUrl ?? filament()->getTenantProfileUrl()"
-                    :target="($profileItem?->shouldOpenUrlInNewTab() ?? false) ? '_blank' : null"
                     :icon="$profileItem?->getIcon() ?? 'heroicon-m-cog-6-tooth'"
+                    icon-alias="panels::tenant-menu.profile-button"
```

Comment:
I think this would break being able to set an icon on the MenuItem instance, as getIcon() would be ignored

Maybe, instead of using this attribute, we need to resolve the icon after ?? and use it as the default

Reply:
Oh, I just copied this over from the user menu, but I think you're right.